### PR TITLE
Add interface to request mission details.

### PR DESCRIPTION
**Pull request by pmtischler**
For #148

### CODE REVIEW

File: server/auvsi_suas/views/missions.py [hunk 1]
```diff
@@ -76,17 +78,33 @@ def mission_for_request(request_params):
...
+    """Handles requests for a specific mission."""
+
+    @method_decorator(require_login)
+    def dispatch(self, *args, **kwargs):
+        return super(MissionsId, self).dispatch(*args, **kwargs)
+
+    def get(self, request, pk):
+        try:
+            mission = MissionConfig.objects.get(pk=pk)
+            return HttpResponse(
```
Comment (prattmic) [inline server/auvsi_suas/views/missions.py]: JsonResponse

File: server/auvsi_suas/views/missions_test.py [hunk 2]
```diff
@@ -107,67 +109,65 @@ def test_not_authenticated(self):
...
-
     def test_post(self):
         """POST not allowed"""
+        self.Login(is_superuser=False)
         response = self.client.post(missions_url)
         self.assertEqual(405, response.status_code)
 
     def test_no_missions(self):
         """No missions results in empty list."""
+        self.Login(is_superuser=True)
```
Comment (prattmic) [inline server/auvsi_suas/views/missions_test.py]: Why are these superusers but the one above is not?

**Comment by prattmic** [comment]
A couple comments, otherwise LGTM

File: server/auvsi_suas/views/missions_test.py (hunk 2 above)
Comment (pmtischler) [inline server/auvsi_suas/views/missions_test.py]: No reason, just a mistake. Fixed.

File: server/auvsi_suas/views/missions.py (hunk 1 above)
Comment (pmtischler) [inline server/auvsi_suas/views/missions.py]: Done.

**Pull request merged**